lib/puppet/pops/types/puppet_object.rb

```diff
@@ -11,6 +11,16 @@ module PuppetObject
   def _ptype
     self.class._ptype
   end
+
+  def _pall_contents(path, &block)
+  end
+
+  def _pcontents
```

Comment:
not that happy with the name as it becomes part of the API of every AST node. Suggested `_pcore_contents` elsewhere.